server/db/controllers/Message.js

```diff
@@ -0,0 +1,174 @@
+import dotenv from 'dotenv';
+import ST from '../../helpers/status';
+import MSG from '../../helpers/res_messages';
+import { CREATE_MESSAGE, 
+  CREATE_INBOX, 
+  CREATE_SENT,
+  GET_USER , 
+  GET_RECEIVED_MESSAGES,
+  GET_UNREAD_RECEIVED_MESSAGES,
+  GET_SENT_RECEIVED_MESSAGES,
+  GET_SPECIFIC_MESSAGES,
+  DELETE_MESSAGES} from '../helpers/query'
```

Comment:
Missing semi-colon at the end of line 12. Actually the part of this line starting from `}` should be on the next line.